README.md

```diff
@@ -97,6 +97,17 @@ This setup exposes:
 - **Port 13306** for MySQL wire protocol connections.
 - **Port 15432** for PostgreSQL wire protocol connections, allowing direct DuckDB SQL.
 
+To initialize MyDuck Server by executing custom SQL statements once it’s ready, you can place the SQL statements in a file with the .sql extension and mount the file to either `/docker-entrypoint-initdb.d/mysql/` or `/docker-entrypoint-initdb.d/postgres/` in the Docker container. The choice between these directories depends on the protocol used for executing the SQL statements.
```

Comment:
This paragraph should be added as a separate subsection and be placed after the "Replicating Data" subsection.

Reply:
done